public/app/config/env/development.js

```diff
@@ -29,6 +29,11 @@
 
             return function () {
                 container.register('options.service', options);
+                container.register('options.providence', {
+                    baseUrl: {
+                        objects: 'https://staging-collections.histwest.org.au/index.php/editor/objects/ObjectEditor/Summary/'
```

Comment:
This matches the service definition options further up in the file; development frontend -> staging CA